examples/resources/hpegl_vmaas_load_balancer/nsx_t_lb.tf

```diff
@@ -1,15 +1,16 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_load_balancer" "tf_load_balancer" {
-  name  = "lab-1"       
+  name  = "tf_LB"  
   description  = "creating load balancer for test"
-  enabled      =    true        
-  visibility   = "public"
+  enabled      =    true      
   resource_permission {
     all = true
   }
   config {
+    log_level = "Info"
```

Comment:
just a doubt, are these case sensitive?

Reply:
yes its case sensitive i will update

Reply:
Do we need validation or parsing for safety?

Reply:
validation already added and we will get the error  "msg": "Field level validation errors: {value small of property size is not one of the allowed values [SMALL, MEDIUM, LARGE, XLARGE, DLB]}"